tests/test-cases/demo-stacks.yaml

```diff
@@ -150,7 +150,7 @@ tests:
         - "^$"
       exit_code: 0
   - name: atmos_vendor_pull
```

Comment:
Move this to `tests/test-cases/vendor-test.yaml`, because this test has nothing to do with `../examples/demo-stacks`